rotkehlchen/api/v1/schemas.py

```diff
@@ -4065,3 +4065,32 @@ def validate_schema(
                         message=f'Account {address} with chain {evm_chain.to_name()} is not tracked by rotki',  # noqa: E501
                         field_name='address',
                     )
+
+
+class AddressesInteraction(Schema):
+    from_address = fields.String(required=True)
+    to_address = fields.String(required=True)
+
+
+class AssetTransferSchema(AddressesInteraction):
+    amount = PositiveAmountField()
+
+
+class TokenTransfer(AssetTransferSchema):
+    token = AssetField(expected_type=EvmToken)
+
+
+class NativeAssetTransfer(AssetTransferSchema):
+    blockchain = BlockchainField(required=True)
```

Comment:
```suggestion
    blockchain = BlockchainField(required=True, limit_to=SUPPORTED_EVM_CHAINS)
```

this way you don't need the validation below